js/src/index.ejs

```diff
@@ -11,28 +11,23 @@
         height: 100%;
         margin: 0;
         padding: 0;
-      }
-
-      .loading-container {
-        display: flex;
-        align-items: center;
-        justify-content: center;
-        height: 100%;
-        font-family: Roboto;
-        background-color: rgba(0, 0, 0, 0.8);
-        color: #ddd;
+        background: white;
+        font-family: 'Roboto', sans-serif;
+        font-size: 16px;
```

Comment:
Material-UI default to 16px, so sticking with that so there is no jarring effect. (Bear in mind that this gets replaced in some dapps with "Loading Application" as soon as it does it's thing, the dapps are also at 2em, for the loading font)